### PR TITLE
Make sure the karate tests produces good reports

### PR DESCRIPTION
**Pull request by GabrielFleischer**
This PR review the report system to make sure readable and useable reports are produced

### CODE REVIEW

**Review by Warhedgehog — CHANGES_REQUESTED** [review]
Nice work, just the little comment error afaiac

File: tests/karate/src/test/java/be/BackEndTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      .relativeTo(getClass())
+      .karateEnv("scala");
+  }
+
+  /**
+   * This is the main front-end test, it is made to be executed from the command line :
+   * <p>
+   * <code>mvn test -DargLine=-Dkarate.env=env -Dtest=BackEndTest#fullTest</code>
+   * <p>
+   * It will execute all frontend tests and generate a report
```
[the remark this answers is not in the document]
Comment (GabrielFleischer) [reply tests/karate/src/test/java/be/BackEndTest.java:44]: Yes 😅

Comment (GabrielFleischer) [inline tests/karate/src/test/java/be/BackEndTest.java:44]:
```suggestion
   * It will execute all backend tests and generate a report
```

**Review by Warhedgehog — APPROVED** [review]
LGTM

**Pull request merged**